test/k8sT/manifests/demo_ds.yaml

```diff
@@ -209,6 +209,7 @@ metadata:
   name: test-lb
 spec:
   type: LoadBalancer
+  loadBalancerIP: 192.168.36.240
```

Comment:
this field should be populated by the LB. Also, different infrastructures will have different IPs.

Reply:
dummyLB will set that IP in the status. The IP is set from the same range as we used before with MetalLB.

Reply:
https://github.com/cilium/dummylb/blob/master/main.go#L39